packages/abi/package.json

```diff
@@ -17,11 +17,12 @@
   "scripts": {
     "docs": "typedoc",
     "prebuild": "rimraf lib",
-    "build": "tsc",
+    "build": "yarn && ./node_modules/.bin/tsc",
```

Comment:
On my machine there's no need for this addition. Did you get an error?

Reply:
Yes, I was getting the following errors:
```
@parity/abi: /bin/sh: tsc: command not found
@parity/electron: /bin/sh: tsc: command not found 
```
In the details of the commit message I included the log https://github.com/paritytech/js-libs/pull/90/commits/73c2042822fa558d4f29a3fcbf4151bdc278db3f

Reply:
These errors are in the fether, right? If yes, then see my comment about removing step 7